cmake/projects/odb-compiler/hunter.cmake

```diff
@@ -29,9 +29,9 @@ elseif(CMAKE_HOST_UNIX)
       VERSION
       "2.4.0"
       URL
-      "http://www.codesynthesis.com/download/odb/2.4/odb-2.4.0-i686-linux-gnu.tar.bz2"
```

Comment:
> to the Raspberry Pi with no problems with the current design

This is because there are no conflicts yet. I'm talking that in general such approach will not work correctly. For example you have a tool `mytool` which should be run on Linux host which depends on `libz.so` shared library. And you want to use shared zlib in your Android project which is also will be installed with name `libz.so` to the same location. Solution is to have separate host and target install directories.